### ISSUE TITLE
RSA+SHA224 sigalg removed from master

### ISSUE THREAD

**Issue by raycoll** [openssl/openssl#3633]
Hello,

We use the `s_client` app for TLS integration tests against our server implementation. When using `s_client` compiled Openssl master, I noticed the `RSA+SHA224` sigalg is no longer available.  A git-bisect shows 703bcee021 introduced this, but doesn't mention removing sha224. Was this sigalg intentionally removed?  If it was intentional(probably because it is removed in TLS1.3 and rarely in existing TLS systems) it is worth calling it out somewhere in the docs.

```
# s_client failure
> ./libcrypto-root-master/bin/openssl version
OpenSSL 1.1.1-dev  xx XXX xxxx
> ./libcrypto-root-master/bin/openssl s_client -connect s3.amazonaws.com:443 -sigalgs "RSA+SHA224"
Error with command: "-sigalgs RSA+SHA224"
```

Thanks

**Comment by richsalz** [comment]
No, that commit is unrelated. 
It worked in 1.0.2?  It should not have been removed in 1.1.  ping @mattcaswell

**Comment by raycoll** [comment]
Works in latest 1.0.2 and 1.1.0 releases:

```
> /usr/local/openssl-1.1.0f/bin/openssl version 
OpenSSL 1.1.0f  25 May 2017
 echo "Q" | /usr/local/openssl-1.1.0f/bin/openssl s_client -connect s3.amazonaws.com:443 -sigalgs "RSA+SHA224"
CONNECTED(00000005)
...
Peer signing digest: SHA224
Server Temp Key: ECDH, P-256, 256 bits
...
New, TLSv1.2, Cipher is ECDHE-RSA-AES128-GCM-SHA256
DONE

> /usr/local/openssl-1.0.2l/bin/openssl version
OpenSSL 1.0.2l  25 May 2017
> echo "Q" | /usr/local/openssl-1.0.2l/bin/openssl s_client -connect s3.amazonaws.com:443 -sigalgs "RSA+SHA224"
CONNECTED(00000003)
...
Peer signing digest: SHA224
Server Temp Key: ECDH, P-256, 256 bits
New, TLSv1/SSLv3, Cipher is ECDHE-RSA-AES128-GCM-SHA256
...
DONE
```

**Comment by mattcaswell** [comment]
Fix in #3639

**Comment by raycoll** [comment]
Thanks!
